variants/tlora_t3s3_epaper/platformio.ini

```diff
@@ -0,0 +1,23 @@
+[env:tlora-t3s3-epaper]
+extends = esp32s3_base
+board = tlora-t3s3-v1
+board_check = true
+upload_protocol = esptool
+
+build_flags = 
+  ${esp32_base.build_flags} -D TLORA_T3S3_EPAPER -I variants/tlora_t3s3_epaper
+  -DGPS_POWER_TOGGLE
+  -DEINK_DISPLAY_MODEL=GxEPD2_213_BN
+  -DEINK_WIDTH=250
+  -DEINK_HEIGHT=122
+  -DUSE_EINK_DYNAMICDISPLAY            ; Enable Dynamic EInk
+  -DEINK_LIMIT_FASTREFRESH=10          ; How many consecutive fast-refreshes are permitted
+  -DEINK_LIMIT_RATE_BACKGROUND_SEC=30  ; Minimum interval between BACKGROUND updates
+  -DEINK_LIMIT_RATE_RESPONSIVE_SEC=1   ; Minimum interval between RESPONSIVE updates
+;   -D EINK_LIMIT_GHOSTING_PX=2000      ; (Optional) How much image ghosting is tolerated
+  -DEINK_BACKGROUND_USES_FAST          ; (Optional) Use FAST refresh for both BACKGROUND and RESPONSIVE, until a limit is reached.
+  -DEINK_HASQUIRK_GHOSTING             ; Display model is identified as "prone to ghosting"
+  -DEINK_HASQUIRK_WEAKFASTREFRESH      ; Pixels set with fast-refresh are easy to clear, disrupted by sunlight
```

Comment:
I'd maybe go easy on some of these values. I don't have this device, but I believe it's the same display as in Heltec Wireless Paper V1.1. From memory, the fast-refresh operation right now is quite vicious. It'd definitely consider using the same profile as with Wireless Paper V1.1:
```
  -D USE_EINK_DYNAMICDISPLAY            ; Enable Dynamic EInk
  -D EINK_LIMIT_FASTREFRESH=5           ; How many consecutive fast-refreshes are permitted
  -D EINK_LIMIT_RATE_BACKGROUND_SEC=30  ; Minimum interval between BACKGROUND updates
  -D EINK_LIMIT_RATE_RESPONSIVE_SEC=1   ; Minimum interval between RESPONSIVE updates
  ;-D EINK_BACKGROUND_USES_FAST         ; Disabled, so display will perform a full refresh once user input ceases
  -D EINK_HASQUIRK_VICIOUSFASTREFRESH   ; Identify that pixels drawn by fast-refresh are harder to clear

  ; I don't remember how bad the image ghosting is, but you might need this
  ;-D EINK_LIMIT_GHOSTING_PX=2000
  ```
  I don't think I made any attempt to optimize the fast-refresh operation for this display. It's using the values the official GxEPD2 library uses. It might be possible room to get a bit more performance out of the panel with some careful adjustments.

Reply:
I didn't see any ghosting and the fastrefresh=10 looks fine to me. I copied from the Heltec E213 (it's PC1 not BN), but let me try your suggestion if I can spot any difference...

Reply:
It's definitely been a few months since I've powered on that older Wireless Paper model, so I don't remember exactly how rough things were. I just have memories of it tolerating fast refresh fairly poorly is all. It's also not impossible that the panel used with this T3S3 model is a slightly different revision too, with different characteristics, so will have to defer to your first-hand experience with it!

Reply:
Removing `EINK_BACKGROUND_USES_FAST` will also use full-refresh for any low priority display updates, which is another way to potentially preserve the display health, if it seems like it might be an issue.

Reply:
Uuuh, it got much worse and is crashing now and booting in a loop
`DEBUG | ??:??:?? 72 [RadioIf] SX126x standby failed with error -707`

Reply:
As a result of changing the profile? I definitely don't know why it would have had any impact like that. Those macros just influence some high-level decision making in when to use full vs fast refresh. Nothing at a driver level.

Reply:
I've updated the defines. Looks ok now.